### PR TITLE
Merge develop to master

### PR DESCRIPTION
**Pull request by elviani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/configuration/references.ts [hunk 1]
```diff
@@ -50,6 +50,9 @@ export const FEATURE_TERMS_AND_CONDITIONS_ENABLED = 'termsAndConditionsEnabled'
 export const FEATURE_HELMET_ENABLED = 'helmetEnabled'
 export const FEATURE_REDIS_ENABLED = 'redisEnabled'
 
+export const LINKS_MANAGE_CASES_LINK = 'links.manageCaseLink'
```
Comment (udaydenduluri33) [inline api/configuration/references.ts:53]: We dont need Manage Cases for MO. Can we remove it?

File: api/configuration/ui.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ import {
   FEATURE_SECURE_COOKIE_ENABLED,
   IDAM_CLIENT,
   INDEX_URL,
+  LINKS_MANAGE_CASES_LINK,
```
Comment (udaydenduluri33) [inline api/configuration/ui.ts:14]: Same here, can remove Manage cases

File: api/configuration/references.ts (hunk 1 above)
Comment (elviani) [reply api/configuration/references.ts:53]: No We need both link in the MO register page

File: api/configuration/ui.ts (hunk 2 above)
Comment (elviani) [reply api/configuration/ui.ts:14]: we need to display the MC link in the register page

**Pull request merged**